app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -79,6 +83,22 @@
 
     /**
      * @param input The {@link EditText} to evaluate.
+     * @return The resultcode of validateIban.
+     */
+    Constants.InputFieldResultCodes validateIban(final EditText input) {
+        final String iban = input.getText().toString();
```

Comment:
When you closed an earlier PR of this branch you commented on my Validate suggestions if we were already done discussing validations. I think that we should validate our user input, not just because we got whole lectures about checking the class invariant, pre-conditions, post-conditions, etc. but also because it gives you much more safety because you're sure that your input meets certain requirements (i.e. it's literally impossible to get nullPointerExceptions if you validate your input)